docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
+MAINTAINER Rodney Gomes (rodney@jut.io)
+
+CMD outriggerd
+
```

Comment:
I find the whole `EXPOSE` thing to be odd to say the least as you can see by the documentation: ```The EXPOSE instruction informs Docker that the container listens on the specified network ports at runtime. EXPOSE does not make the ports of the container accessible to the host. To do that, you must use either the -p flag to publish a range of ports or the -P flag to publish all of the exposed ports`` so basically adding the EXPOSE still means you have to use the -P flag but you're basically saying with -P is expose everything the image wants without caring about how its exposed. Either way I can put the expose and change the documentation to say use `-P`.